sys/src/cmd/cc/lex.c

```diff
@@ -60,12 +60,21 @@ main(int argc, char *argv[])
 		break;
 
 	case 'l':			/* for little-endian mips */
-		if(thechar != 'v'){
-			print("can only use -l with vc");
+		switch(thechar) {
```

Comment:
ah, why is this? I am now lost. The commit message is not helping at all.

Reply:
if you diff 4c/lex.c and cc/lex.c you'll see this is the main difference, so i moved it to cc/lex.c.

of course this could all go away if we can agree mips is dead....